src/App.tsx

```diff
@@ -46,54 +55,46 @@ const App: React.FC = () => {
     | FrameContexts
     | undefined;
 
-  const renderContent = (
-    isExtensionReady: boolean,
-    frameContext?: FrameContexts
-  ) => {
-    if (!isExtensionReady) {
-      return (
-        <Flex vAlign="center" hAlign="center" style={{ height: "100vh" }}>
```

Comment:
**question (non-blocking)**: Is this step (showing the loading spinner) no longer needed during the initialization of the extension?

Reply:
It initializes so fast that most of the time, we can't even notice the spinner.